### PR TITLE
Graphical abstract and sequence features figure

### PR DESCRIPTION
**Pull request by jjc2718**
Added an initial version of the graphical abstract. It currently contains all 3 model types in a single figure, but I also have the model types in separate SVG images in case we'd prefer to separate them and use one as the graphical abstract.

Also added a figure depicting the difference between "hand-engineered" sequence features and sequence convolutions.

### CODE REVIEW

**Comment by cgreene** [comment]
Did you want to include the other figures somewhere else in the text & their legends?

**Comment by jjc2718** [comment]
Sure, if you think it's okay to duplicate them. I guess we can always remove them if we need to.

**Comment by cgreene** [comment]
I think it's important to have them here with figure legends and not just in the graphical abstract. I'm guessing that the response is going to be a request that we come up with a simpler abstract, but we'll see 😁

File: content/04.sequence-models.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 CNNs are a neural network variant that groups input data by spatial context to extract features for prediction.
 
 The definition of "spatial context" is specific to the input: one might group image pixels that are nearby in 2D space, or genomic base pairs that are nearby in the linear genome.
 In this way, CNNs consider context without making strong assumptions about exactly how much context is needed or how it should be encoded; the data informs the encoding.
 A detailed description of how CNNs are applied to sequences can be found in Angermueller et al. [@doi:10.15252/msb.20156651].
 
+![Depiction of contrasting approaches to extracting features from DNA or RNA sequence data. Early models defined features of interest by hand based on prior knowledge about the prediction or clustering problem of interest, such as GC content or sequence melting point. Convolutional models use sequence convolutions to derive features directly from sequence proximity, without requiring quantities of interest to be identified before the model is trained.](images/sequence_features.svg){#fig:sequence_features .white}
```
Comment (cgreene) [inline content/04.sequence-models.md]:
```suggestion
![Contrasting approaches to extracting features from DNA or RNA sequence data. Early models defined features of interest by hand based on prior knowledge about the prediction or clustering problem of interest, such as GC content or sequence melting point. Convolutional models use sequence convolutions to derive features directly from sequence proximity, without requiring features of interest to be identified before the model is trained.](images/sequence_features.svg){#fig:sequence_features .white}
```

File: content/05.network-models.md [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
-To account for relationships between genes, one might incorporate known interactions or correlations when making predictions or generating a low-dimensional representation of the data.
+To account for relationships between genes, one might incorporate known interactions or correlations when making predictions or generating a low-dimensional representation of the data (Figure {@fig:network_models}).
 This is comparable to the manner in which sequence context pushes models to consider nearby base pairs similarly.
 
+![When applying machine learning to gene expression data, it can be helpful to incorporate structure describing the relationships between genes. One common approach is to use a network or collection of gene sets to embed the data in a lower-dimensional space, in which genes that are in the same gene sets or that are well-connected in the network have a similar representation in the lower-dimensional space. The embedded data can then be used for classification or clustering tasks.](images/network_models.svg){#fig:network_models .white}
```
Comment (cgreene) [inline content/05.network-models.md]:
```suggestion
![The relationships between genes provide structure that can be incorporated into machine learning models. One common approach is to use a network or collection of gene sets to embed the data in a lower-dimensional space, in which genes that are in the same gene sets or that are well-connected in the network have a similar representation in the lower-dimensional space. The embedded data can then be used for classification or clustering tasks.](images/network_models.svg){#fig:network_models .white}
```

File: content/06.other-models.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ DeepGO [@doi:10.1093/bioinformatics/btx624] uses a similar approach to predict p
 However, a follow-up paper by the same authors [@doi:10.1093/bioinformatics/btz595] showed that this hierarchy-aware approach can be outperformed by a hierarchy-naive CNN, which uses only amino acid sequence and similarity to labeled training set proteins.
 This suggests a tradeoff between interpretability and predictive accuracy for protein function prediction.
 
+![One approach to incorporating directed graph-structured data (such as an ontology or phylogenetic tree) into machine learning models is to use it to inform the structure of a neural network. Here, the connections in the neural network used to predict a set of labels parallel those in the tree graph. This type of constraint can also be useful in model interpretation: for example, if the red-shaded nodes have high neuron outputs for a given sample, then the subsystem encoded in the red-shaded part of the tree graph is most likely important in making predictions for that sample.](images/ontology_models.svg){#fig:ontology_models .white}
```
Comment (cgreene) [inline content/06.other-models.md]:
```suggestion
![Directed graph-structured data, such as an ontology or phylogenetic tree can be incorporated into machine learning models. Here, the connections in the neural network used to predict a set of labels parallel those in the tree graph. This type of constraint can also be useful in model interpretation: for example, if the red-shaded nodes have high neuron outputs for a given sample, then the subsystem encoded in the red-shaded part of the tree graph is most likely important in making predictions for that sample.](images/ontology_models.svg){#fig:ontology_models .white}
```

**Pull request merged**